draft-thomson-webpush-protocol.xml

```diff
@@ -328,7 +338,40 @@ Cache-Control: max-age:864000, private
 ]]></artwork>
       </figure>
     </section>
+    <section anchor="receipt_subscription" title="Subscribing to Receipts">
+      <t>
+        An application server sends a POST request to its configured push
```

Comment:
As for the explanation: if you accept the status quo, an application server who consumes the web push API needs to develop a relationship with every possible push service that their users might use.  That doesn't scale.

There's nothing inherently wrong with developing a relationship and using that to receive some different treatment.  Perhaps random passersby are subject to strict DoS-mitigation limits on message sending that might make your application infeasible.  For instance, I'm sure that large site operators will want to develop a special relationship with most of the big push operators if only to avoid being throttled unnecessarily.